source/includes/_datahub.md

```diff
@@ -0,0 +1,678 @@
+# Introduction
+
+Bullhorn Data Hub is a flexible data structure to store data for reporting and analytics across Bullhorn, Marketplace Partners and external sources. It is built directly within the Bullhorn ATS Platform to help customers consolidate their staffing and relevant business data in a single location. This reference guide provides information about each Data Hub API operation and supported entity type.
+
+# Data Hub Entities
+
+## EdsSourceSystem
+Represents the external system sending data. This entity's name is represented as `sourceSystem` when making Data Hub API requests. See [POST / PUT /data-hub/sourceSystem](https://bullhorn.github.io/rest-api-docs/datahubref.html#post-put-data-hub-sourcesystem) section of Data Hub API for more details.
+
+<table>
+    <colgroup>
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+    </colgroup>
+    <thead>
+        <tr class="header">
+            <th>SourceSystem field</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Not null</th>
+            <th>Read-only</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr class="odd">
+            <td>name</td>
+            <td>String(64)</td>
+            <td>Name of the source system.</td>
+            <td>X</td>
+            <td></td>
+        </tr>
+        <tr class="even">
+            <td>display</td>
+            <td>String(64)</td>
+            <td>How the source system will be displayed.</td>
+            <td>X</td>
+            <td></td>
+        </tr>
+    </tbody>
+</table>
+
+## EdsEntityType
+Represents a category of data in the Source System. This entity's name is represented as `entityType` when making Data Hub API requests. See [POST / PUT /data-hub/entityType](https://bullhorn.github.io/rest-api-docs/datahubref.html#post-put-data-hub-entitytype) section of Data Hub API for more details.
```

Comment:
I would lower case things like Source System, Entity Type, Entity Name etc. Anything that's not Data Hub and isn't and is a generic name.

Reply:
Updated